app/models/manageiq/providers/openshift/container_manager/refresher.rb

```diff
@@ -3,9 +3,24 @@ module Openshift
     class ContainerManager::Refresher < ManageIQ::Providers::BaseManager::Refresher
       include ::EmsRefresh::Refreshers::EmsRefresherMixin
 
+      def self.entities
+        %w(routes projects)
+      end
+
       def parse_inventory(ems, _targets = nil)
-        all_entities = ems.with_provider_connection(&:all_entities)
-        all_entities.merge!(ems.with_provider_connection(:service => ManageIQ::Providers::Kubernetes::ContainerManager.ems_type, &:all_entities))
+        all_entities = ems.with_provider_connection { |client| fetch_entities(client, self.class.entities) }
+        fetch = proc do |client|
+          fetch_entities(
+            client,
+            ManageIQ::Providers::Kubernetes::ContainerManager::Refresher.entities
+          )
+        end
+        all_entities.merge!(
+          ems.with_provider_connection(
+            :service => ManageIQ::Providers::Kubernetes::ContainerManager.ems_type,
+            &fetch
```

Comment:
This does not work for some reason, the code runs fine but openshift entities are not fetched.
seems scope related.
```
kubernetes_entities = ems.with_provider_connection(:service => KUBERNETES_EMS_TYPE) do |client|
  fetch_entities(client, self.class.entities)
end
```